scm/flags.go

```diff
@@ -67,4 +67,16 @@ var Flags = []cli.Flag{
 			"is behind a Firewall or NAT, or when using something like ngrok to forward webhooks. " +
 			"(defaults to VELA_ADDR).",
 	},
+	&cli.Int64Flag{
+		EnvVars:  []string{"VELA_SCM_APP_ID", "SCM_APP_ID"},
+		FilePath: "/vela/scm/app_id",
+		Name:     "scm.app.id",
+		Usage:    "set ID for the SCM App integration (GitHub App)",
+	},
+	&cli.StringFlag{
+		EnvVars:  []string{"VELA_SCM_APP_PRIVATE_KEY", "SCM_APP_PRIVATE_KEY"},
+		FilePath: "/vela/scm/app_private_key",
+		Name:     "scm.app.private_key",
+		Usage:    "set value of base64 encoded SCM App integration (GitHub App) private key",
```

Comment:
should we make this "path to private key" instead of the value? would make it easier to drop files instead of env vars

Reply:
I think that is definitely more standard

Reply:
done! heres a conversion one liner if you want to test using the key you might already have in `.env`
 
```
( source .env; echo -n "$VELA_SCM_APP_PRIVATE_KEY" |  base64 -d ) >> .privatekey && ( echo "VELA_SCM_APP_PRIVATE_KEY_PATH=.privatekey" >> .env && sed -i~ '/^VELA_SCM_APP_PRIVATE_KEY=/s/=.*/=""/' .env && rm .env~)
```